DigitalLearningSolutions.Web/Views/Shared/SearchablePage/_PaginationInputs.cshtml

```diff
@@ -0,0 +1,15 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.Common.SearchablePage

```

Comment:
I think this probably needs a different name, this implies to me it is the inputs that are required for the pagination functionality, not the other inputs for the rest of the stuff.